<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1138</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Serwer: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ServerInput"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="PortInput"/>
        </item>
        <item>
         <widget class="QPushButton" name="ConnectButton">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Nickname:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Nickname">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setNickname">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Set Nickname</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="channels">
        <property name="title">
         <string>Channels</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="channelsElements"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="topMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Message: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="MessageInput"/>
        </item>
        <item>
         <widget class="QPushButton" name="SendMessage">
          <property name="text">
           <string>Send to channel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Command:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="CommandInput"/>
        </item>
        <item>
         <widget class="QPushButton" name="SendCommand">
          <property name="text">
           <string>Send command</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Users:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="usersList"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Channels:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createChannelButton">
        <property name="text">
         <string>Create channel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="channelsList"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1138</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
